src/python/pants/backend/jvm/tasks/run_jvm_prep_command.py

```diff
@@ -50,7 +49,6 @@ def prepare(cls, options, round_manager):
     """
     :API: public
     """
-    super().prepare(options, round_manager)
```

Comment:
If this is intentional, it would be good to add a comment explaining why it is necessary.

Reply:
It's intentional. The reason I did it was because some jvm prep command tasks may not need the 'runtime_classpath', but JvmTask always requests it.

Looking at it again, I don't think this change is necessary, it just means that the next patch will need to modify prep command too, which isn't a huge change.